### DISCUSSION TITLE
BuildHow to Start Creating VPN Client on Android

### DISCUSSION THREAD

**Discussion by t-51** [strongswan/strongswan#2198] [Q&A]
- Hi. - Good time. 
- I want to build a VPN or use StrongSwan but I don't know where to start and I don't understand it correctly. 
- Can you please guide me?

**Comment by tobiasbrunner** [comment]
Maybe the docs on [how to build our Android app](https://docs.strongswan.org/docs/5.9/os/androidVpnClientBuild.html) helps?

**Reply by t-51** [reply]
Thank you very much, I just have a question, can it be implemented with Kotlin without any problems?

**Reply by tobiasbrunner** [reply]
As far as I know, you can mix Kotlin and Java code without issues.

**Reply by t-51** [reply]
Thank you، I have found and imported the files. 
But the question is, I don't have a complete documentation of how to use these classes and what each class does! 
Please guide me، I would be grateful.
